Library/Formula/duplicity.rb

```diff
@@ -13,14 +12,9 @@ class Duplicity < Formula
     sha256 "e5f3f41cbb71d4e543c1de7a6aa7b81bef0c395e60ab459e46e35cea54b66fd7" => :mountain_lion
   end
 
-  devel do
-    url "https://code.launchpad.net/duplicity/0.7-series/0.7.05/+download/duplicity-0.7.05.tar.gz"
-    sha256 "1e53807d5de5bacbbe6c4eea2f313a16e9a63423316ae7b8597ee0d9e697f744"
-  end
-
   option :universal
 
-  depends_on :python if MacOS.version <= :snow_leopard
+  depends_on :python
```

Comment:
This shouldn't be made unconditional, otherwise people will have to install brewed Python to use it.